README.md

```diff
@@ -23,16 +23,17 @@ First-of-its-kind patent-pending technology that automatically correlates outbou
 2. In the workflow logs, you will see a link to security insights and recommendations.  
 
     <p align="left">
-      <img src="https://step-security-images.s3.us-west-2.amazonaws.com/build_log_link1.png" alt="Link in build log" >
+      <img src="https://github.com/step-security/supply-chain-goat/blob/main/images/harden-runner/ActionLog.png" alt="Link in build log" >
     </p>
 
 3. Click on the link ([example link](https://app.stepsecurity.io/github/jauderho/dockerfiles/actions/runs/1736506434)). You will see outbound traffic made by each step.
 
     <p align="left">
-      <img src="https://step-security-images.s3.us-west-2.amazonaws.com/insights3.png" alt="Insights from harden-runner" >
+      <img src="https://github.com/step-security/supply-chain-goat/blob/main/images/harden-runner/OnlineTool.png" alt="Insights from harden-runner" >
```

Comment:
@arjundashrath similar feedback as for earlier one. the outbound calls can be highlighted, we do not need to show all actions, just few top ones.